.github/workflows/pull-request-workflow.yml

```diff
@@ -62,3 +62,33 @@ jobs:
         run: yarn install --immutable
       - name: Build
         run: yarn build
+
+  tests:
+    needs:
+      - build
+    runs-on: ubuntu-latest
+    container: node:16-alpine3.15
+    services:
+      postgres:
+        image: postgres
+        env:
+          POSTGRES_PASSWORD: example
+          POSTGRES_DB: test_integration
+        options: >-
+          --health-cmd pg_isready
+          --health-interval 10s
+          --health-timeout 5s
+          --health-retries 5
```

Comment:
Can you explain why do we need to these options?

Reply:
I followed GitHub Actions documentation.

https://docs.github.com/en/actions/using-containerized-services/creating-postgresql-service-containers#running-jobs-directly-on-the-runner-machine

Reply:
Okay, but documentation isn't always show you what you need exactly. It's ambiguous to make a blind copy without knowing of each parameter and explanation why do we need this. So, I wait for your explanation why do we need to keep this

Reply:
We need to be sure that our database is running, and our application can communicate with it correctly.